app/assets/stylesheets/govuk-component/_previous-and-next-navigation.scss

```diff
@@ -1,81 +1,83 @@
-.govuk-previous-and-next-navigation {
+.pub-c-pagination {
   display: block;
   margin-top: $gutter;
   margin-bottom: $gutter;
   margin-left: -$gutter-half;
   margin-right: -$gutter-half;
```

Comment:
I prefer long hand since it's super obvious what's happening but I think I'm in the minority.

I compare it to options objects passed to a function

`function render('potato', true, false, true, false)`

`function render('potato', {
  upsideDown: true,
  invisible: false,
  superCool: true
})`